containers/concent-api/install-concent-api.sh.j2

```diff
@@ -6,6 +6,7 @@ virtualenv_dir=/srv/http/virtualenv
 virtualenv --python "python{{ python_version }}" "$virtualenv_dir"
 source "$virtualenv_dir/bin/activate"
 python -m pip install --upgrade pip
+cd /srv/http/concent_api/
```

Comment:
Please add a comment that requirements needs to be installed from the directory it's in because it contains relative paths.

Reply:
Done.